utils/providers.py

```diff
@@ -145,7 +149,6 @@ def setup_a_provider(prov_class=None, prov_type=None, validate=True, check_exist
     """
     global problematic_providers
     if prov_class == "infra":
-        from cfme.infrastructure.provider import get_from_config
```

Comment:
This and the next three are no longer required now that we have a common function